clouds/snowflake/modules/sql/h3/H3_BOUNDARY.sql

```diff
@@ -1,32 +1,15 @@
 ----------------------------
--- Copyright (C) 2021 CARTO
+-- Copyright (C) 2021-2024 CARTO
 ----------------------------
 
-CREATE OR REPLACE FUNCTION @@SF_SCHEMA@@._H3_BOUNDARY
-(index STRING)
-RETURNS STRING
-LANGUAGE JAVASCRIPT
-IMMUTABLE
-AS $$
-    if (!INDEX) {
-        return null;
-    }
-
-    @@SF_LIBRARY_H3_BOUNDARY@@
-
-    if (!h3BoundaryLib.h3IsValid(INDEX)) {
-        return null;
-    }
-
-    const coords = h3BoundaryLib.h3ToGeoBoundary(INDEX, true);
-    const uniqueCoords = h3BoundaryLib.removeNextDuplicates(coords);
-    return `POLYGON((${uniqueCoords.map(c => `${c[0]} ${c[1]}`).join(',')}))`;
-$$;
-
 CREATE OR REPLACE SECURE FUNCTION @@SF_SCHEMA@@.H3_BOUNDARY
-(index STRING)
+(h3_hex STRING)
```

Comment:
Even if h3_hex sounds more clear I wouldn't update it, as we just use the name `index` in many places so it fits more the current implementation.

If at some point we need to modify parameters names remember that it's also necessary to modify them in the corresponding doc. In this case: https://github.com/CartoDB/carto-analytics-toolbox-core/blob/04bbf15c6a0a4c9c8d14ec7e60af056f94d58cc9/clouds/snowflake/modules/doc/h3/H3_BOUNDARY.md